src/main/java/net/sf/jabref/logic/labelPattern/LabelPatternUtil.java

```diff
@@ -175,8 +175,11 @@ private static String unifyDiacritics(String content) {
      * @return True if the author or editor is an institution.
      */
     private static boolean isInstitution(String author) {
-        return (author.charAt(0) == '{')
-                && (author.charAt(author.length() - 1) == '}');
+        if (!(author.isEmpty())) {
+            return (author.charAt(0) == '{') && (author.charAt(author.length() - 1) == '}');
+        } else {
+            return false; // In case of empty or null author
```

Comment:
As this method will at worst be called with an empty String, I just changed the comment. Also, the same holds for the methods `removeDiacritics()` and `generateInstitutionKey()` so I changed the null test to `isEmpty()` there.